models/dodo/polygon/dodo_aggregator_polygon_trades.sql

```diff
@@ -62,6 +62,30 @@ WITH dexs AS
         {% if is_incremental() %}
         WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
         {% endif %}
+
+        UNION ALL
+        
+        -- DODOFeeRouteProxy
+        SELECT
+            evt_block_time AS block_time,
+            'DODO' AS project,
+            '0' AS version,
+            sender AS taker,
+            '' AS maker,
+            fromAmount AS token_bought_amount_raw,
+            returnAmount AS token_sold_amount_raw,
+            cast(NULL as double) AS amount_usd,
+            fromToken AS token_bought_address,
+            toToken AS token_sold_address,
+            contract_address AS project_contract_address,
+            evt_tx_hash AS tx_hash,
+            '' AS trace_address,
```

Comment:
Hi @owen05, if these parts are adding to dex_aggregator.trades,
we may need to use `array<bigint>` for `trace_address`.

Reply:
Got it. I will change it 👌